### PR TITLE
Upgrade @opentripplanner/trip-details to 2.0.0

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
This PR adds support i18n in the `TripDetails` component.

### CODE REVIEW

**Review by philip-cline — CHANGES_REQUESTED** [review]
Few small comments on formatting, and I think @miles-grant-ibigroup figured out what to do to get the tests passing. 

Use of the number skeletons and special tokens in plurals looks great. I'll update the i18n clean-up PR to use these as well so things are standardized.

**Comment by binh-dam-ibigroup** [comment]
The a11y build should be fixed now thanks to @miles-grant-ibigroup's finding, but awaiting #509 to remove the failing lint-docs step.

File: i18n/en-US.yml [hunk 1]
```diff
@@ -628,6 +628,36 @@ components:
   WelcomeScreen:
     prompt: Where do you want to go?
 
+otpUi:
+  TripDetails:
+    calories: "Calories Burned: <strong>{calories, number, ::.}</strong>"
+    caloriesDescription: "Calories burned is based on
```
[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply i18n/en-US.yml]: Sure. Updated in https://github.com/opentripplanner/otp-react-redux/pull/508/commits/4343bafdece1097102b632748352a11fdcaeaedf.

File: i18n/fr-FR.yml [hunk 2]
```diff
@@ -608,6 +608,37 @@ components:
   WelcomeScreen:
     prompt: Où désirez-vous aller ?
 
+otpUi:
+  TripDetails:
+    calories: "Calories dépensées : <strong>{calories, number, ::.}</strong>"
+    caloriesDescription: "La dépense calorique est calculée sur la base de
```
[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply i18n/fr-FR.yml]: Sure. Updated in https://github.com/opentripplanner/otp-react-redux/pull/508/commits/4343bafdece1097102b632748352a11fdcaeaedf.

**Review by philip-cline — APPROVED** [review]

**Review by daniel-heppner-ibigroup — APPROVED** [review]

**Pull request merged**